vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Install and use packages on Linux
+description: Tutorial guides the user through the process of installing and using packages on Linux with vcpkg.
+author: JavierMatosD
+ms.date: 08/23/2023
```

Comment:
Missing the content metadata tag: ms.topic should be added and set to tutorial.
Here's [the guide on writing tutorials](https://review.learn.microsoft.com/en-us/help/patterns/level4/article-tutorial?branch=main#use-a-template)